tests/lib/snaps.sh

```diff
@@ -43,3 +43,23 @@ install_generic_consumer() {
     snap install --dangerous generic-consumer/*.snap
     rm -rf generic-consumer
 }
+
+is_classic_confinement_supported() {
```

Comment:
I think it'd be better to ask snapd about it, so that we have to maintain less tables. Also this is likely to change a little (in what it really mans to support it) so this may (hopefully) go away.